backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/ListenerApiImpl.scala

```diff
@@ -39,7 +39,7 @@ class ListenerApiImpl extends ListenerApi {
 
   override def onDriverStart(conf: SparkConf): Unit = {
     // sql table cache serializer
-    if (conf.getBoolean(GlutenConfig.COLUMNAR_TABLE_CACHE_ENABLED.key, defaultValue = true)) {
+    if (conf.getBoolean(GlutenConfig.COLUMNAR_TABLE_CACHE_ENABLED.key, defaultValue = false)) {
```

Comment:
I think it can be simplified. The internal default value will be implicitly used if not set.
```
if (conf.getConf(GlutenConfig.COLUMNAR_TABLE_CACHE_ENABLED)) {
...
}
```

Reply:
it is the SparkConf, not the SQLConf...

Reply:
above is a safe way to get conf with bool value, otherwise you need convert. Will left it no change.